index.html

```diff
@@ -7,14 +7,173 @@
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital@0;1&family=Quicksand:wght@300&display=swap"
+        rel="stylesheet">
 
-    <title>Responsive grid project</title>
+    <title> Soraiya Responsive grid project</title>
 </head>
 
 <body>
 
     <!-- Add your markup here -->
 
+    <nav>
+        <div class="container button">
+            <div class="row">
+                <div class="col-2 offset-10"><p>Book an Appointment</p>
+                </div>
+            </div>
+        </div>
+
+        <div class="container title">
+            <div class="row">
+                <div class="col-4"><p>Maria Hope, Ph.D. - Psychologist</p>
+                </div>
+                <div class="col-1 offset-3 nav-link"><a class="#href">Home</a></div>
+                <div class="col-1 nav-link"><a class="#href">About</a></div>
+                <div class="col-1 nav-link"><a class="#href">Services</a></div>
+                <div class="col-1 nav-link"><a class="#href">Blog</a></div>
+                <div class="col-1 nav-link"><a class="#href">Contact</a></div>    
+            </div>
+            </div>
+        </div>
+    </nav>
+    <div class="container main-box">
+        <div class="row align-items-center main-row">
+            <div class="col-6 main"><h1>Online Therapy for Emotional Health</h1>  
+                <div class="row mb-3 arrow-row">
+                    <div class="col-1 offset-6 arrow-column">
+                        <button class="arrow">↓</button>
+                    </div>
+                </div>
+            </div>
+            <div class="col-6 main-img">
+                <img class="succulent-img" src="img/succulent - blog.webp" alt="main img">
+                </div>
+            </div>
+        </div>
+    </div>
+    
+    <div class="container bio-section">
+        <div class="row bio-row">
+            <div class="col-2 offset-5 about-text">
+                <p>About</p>
+            </div>
+        </div>
+        <div class="row bio-name">
+            <div class="col-3 offset-5 name-text">
+                <h2>Maria Hope</h2>
+            </div>
+        </div>
+        <div class="row paragraph">
+            <div class="col-6 offset-3">
+                <p class="small-paragraph">I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to
+                add your own content and make changes to the font. Feel free to drag and drop me anywhere you like on your page. I’m a
+                great place for you to tell a story and let your users know a little more about you.</p>
+            </div>
+        </div>
+        <div class="row mb-3 arrow-row">
+            <div class="col-1 offset-6 arrow-column">
+                <button class="arrow">→</button>
+            </div>
+        </div>
+        <div class="row justify-content-center bio-img-row">
+            <div class="col-5 bio-img-col">
```

Comment:
Trick to get the overlapping image effect:
Add `position: absolute` to this line (line 83)
And then add this to the section:
```
.bio-section {
   margin-bottom: 300px;
   padding-bottom: 400px;
}
    ```
(You might also need to add `z-index: 1;` to your `nav` as the woman's image now sits on top of it)